### ISSUE TITLE
Comment control characters differ between Spec and ABNF

### ISSUE THREAD

**Issue by moorereason** [toml-lang/toml#850]
### Problem Statement

The Spec and ABNF differ in which control characters are permitted in comments.  The Spec says that `U+007F` is not permitted, but the ABNF allows `%x20-7F` (via `non-eol`).

### Spec

The TOML [spec](https://github.com/toml-lang/toml.io/blob/3fa047976650c38b9ef94045803ed48701a09e98/specs/en/v1.0.0.md#comment) makes the following statement:

> Control characters other than tab (U+0000 to U+0008, U+000A to U+001F, U+007F) are not permitted in comments.

### ABNF

The ABNF [comment definition](https://github.com/toml-lang/toml/blob/1.0.0/toml.abnf#L37-L41):

```abnf
comment-start-symbol = %x23 ; #
non-ascii = %x80-D7FF / %xE000-10FFFF
non-eol = %x09 / %x20-7F / non-ascii

comment = comment-start-symbol *non-eol
```

### Solution

Assuming the Spec is correct, I believe `non-eol` should be defined in the ABNF as:

```abnf
non-eol = %x09 / %x20-7E / non-ascii
```

**Comment by pradyunsg** [comment]
A PR fixing this would be very much welcome! :)

**Comment by moorereason** [comment]
Dangit.  I just noticed that it's already fixed in master via #812.

**Comment by moorereason** [comment]
On a related note, can a new release be issued for this issue?
